.yarnrc.yml

```diff
@@ -0,0 +1,5 @@
+yarnPath: '.yarn/releases/yarn-berry.js'
+packageExtensions:
+  'ts-jest@*':
+    dependencies:
+      jest-config: '*'
```

Comment:
[ts-jest imports jest-config](https://github.com/kulshekhar/ts-jest/blob/03903612df09370baee58af41aa9909760c2b987/src/compiler.ts#L36) but [does not depend on it](https://github.com/kulshekhar/ts-jest/blob/03903612df09370baee58af41aa9909760c2b987/package.json#L63-L124). Non-strict `node_modules` layouts are fine allowing this, but yarn's PnP setup is strict. To work around it, we pretend that `ts-jest` does depend on `jest-config`. We should open an issue with `ts-jest`.

Reply:
https://github.com/kulshekhar/ts-jest/issues/1400